shapes/contacts/address-book.shex

```diff
@@ -3,9 +3,22 @@ PREFIX dc:    <http://purl.org/dc/elements/1.1/>
 PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
 PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
 
+# This shape describes some of the information we store for an address book. The
+# resource it is used for is to describe a single address book. We use the id
+# "#this" to refer to the subject of the address book.
```

Comment:
This could use a bit more explanation I think (simple Turtle snippets are always your friend there I think :) !). I assume `#this` is a fragment identifier, and do you mean that you always apply (just as a **_naming convention_**) `#this` to the end of the RDF Subject IRI for AddressBook instances?
But why? Wouldn't that mean I can't dereference that AddressBook instance directly from my browser, i.e., if the instance was `https://pod.com/userX/public/myAddressBook#this`, then my browser will only request `https://pod.com/userX/public/myAddressBook`.
So I think I'm misinterpreting what is meant here by "the subject of the address book".

Reply:
Yeah, you will have to excuse my miswording in this, english is not my native language and I'm a bit out of training wrt using the right, formal words to describe these concepts.

The use of `#this` as a naming convention comes from SolidOS/Data Browser, where `#this` is given special treatment if found in a resource (i.e. it will render the contacts-pane when finding #this and seeing it's of type `vcard:AddressBook`).

Reply:
Ok, no problem at all. The example Turtle files you've add are perfect for demonstrating the usage and intent of `#this`. So I'd just reference the local directory containing those files, something like this:
```suggestion
# "#this" to refer to the subject of the address book (see example Turtle files in ????).
```